### DISCUSSION TITLE
Using Bonsai.Pco Module

### DISCUSSION THREAD

**Discussion by banningjoe** [bonsai-rx/bonsai#2045] [General]
I'm hoping to use the following module (https://github.com/bonsai-rx/pco) but can't find the Bonsai.Pco Package under the package manager. Is this still usable? Thanks!

**Comment by bruno-f-cruz** [comment]
HEya!

I dont think this package was ever released to any feed, probably because its not ready to be used. My advice would be to try to build it in visual studio and check the state of it.
